README.md

```diff
@@ -11,4 +11,10 @@ Python Flask project used to aggregate and distribute IRC meeting minutes and lo
 6. `python3 main.py --help`
 7. `python3 main.py -p 9696 -4`
 
+## How to run using docker
+
+1. Build the docker image `docker build -t <tag> .`
+2. Start the server `docker run -p <docker host port>:9696 <hash of image>`
+3. To start the server on a non default port: `docker run <docker host port>:<container port> <hash of image> -p <container port> -4`
```

Comment:
We will just leave the instructions to sane defaults. For folks who wish to customize their local images with specific ports, I think they would be able to without the need of adding any documentation about it.

Reply:
Seems fine to me, I don't think anyone will have difficulty customizing the ports. I have updated the readme with these changes.
I need to read about about Podman though, that is something new for me.